vlib/v/fmt/tests/struct_embed_input.vv

```diff
@@ -0,0 +1,12 @@
+struct Foo {
+	x int
+}
+
+struct Test {}
+
+struct Bar {
+	y int
+	Foo
+	z string
+	Test
```

Comment:
I think it should be an error if struct embeddings occur after a field declaration.

Reply:
it's not an error in Go, but I think it makes sense to have this, in order to force more readable code